security/compliance_operator/compliance-operator-advanced.adoc

```diff
@@ -14,3 +14,5 @@ include::modules/compliance-raw-tailored.adoc[leveloffset=+1]
 include::modules/compliance-rescan.adoc[leveloffset=+1]
 
 include::modules/compliance-custom-storage.adoc[leveloffset=+1]
+
+include::modules/compliance-apply-remediations-from-scans.adoc[leveloffset=+1]
```

Comment:
I think the following module include is missing from this assembly:
modules/compliance-auto-update-remediations.adoc